README.md

```diff
@@ -22,35 +22,55 @@ A Very Good Infinite List Widget created by [Very Good Ventures][very_good_ventu
 
 ## Usage
 
-A basic `InfiniteList` requires two parameters:
+A basic `InfiniteList` requires four parameters:
 
-- `itemLoader` which is responsible for asynchronously fetching the content
-- `builder` which is responsible for returning a `Widget` given a specific item (result)
+- A list of `items` that are displayed using the `itemBuilder`.
+- An `itemBuilder` that is responsible for returning a widget for every element in `items`.
+- `hasReachedMax` that indicates if any more data is available.
```

Comment:
I think we can remove `hasReachedMax` if we have an `itemCount` as mentioned above because the bottom loading indicator can be driven by whether or not `onFetchData` has completed.